webhook.rst

```diff
@@ -130,3 +130,48 @@ final readonly class WebhookListener implements ConsumerInterface
         // Handle the mail engagement event
     }
 }
+
+Usage in combination with the Notifier component
+------------------------------------------------
+
+The usage of the Webhook component when using a third-party transport in
+the Notifier is very similar to the usage with the Mailer.
+
+Currently, the following third-party sms transports support webhooks:
+
+============ ==========================================
+SMS service  Parser service name
+============ ==========================================
+Twilio       ``notifier.webhook.request_parser.twilio``
+============ ==========================================
+
+.. versionadded:: 6.3
+
+    Webhook support for the Twilio service was introduced in Symfony 6.3.
```

Comment:
Should we keep the same wording as other ones? "The support for Twilio service blablabla..."

Reply:
Yes please

Reply:
Good one. Thanks for that. Updated.